frontend/app/templates/components/transportation/trip-generation-tables/trip-generation-results.hbs

```diff
@@ -50,141 +50,130 @@
+                    {{round (get this.totalsOfWeekdayAutoVehicleTrip time)}}
+                  </td>
+                  <td>
+                    {{#if ta.taxiVehicleOccupancy}}
+                      {{round (get this.totalsOfWeekdayTaxiVehicleTrip time)}}
+                    {{else}}
+                      -
+                    {{/if}}
+                  </td>
+                  <td>
+                    -
+                  </td>
+                  <td>
+                    {{#if ta.taxiVehicleOccupancy}}
+                      {{round (get this.totalsOfTotalsOfWeekdayVehicleTrip time)}}
+                    {{else}}
+                      -
+                    {{/if}}
+                  </td>
+                {{else if (not-eq inOutOrTotal.key "total")}}
+                  {{#each this.modalSplitVariablesSubset as |mode|}}
+                    <td>
+                      {{round (get (get this.saturdayModeCalcs inOutOrTotal.key) mode)}}
+                    </td>
+                  {{/each}}
+                  <td>
+                    {{round (get this.saturdayModesTotals inOutOrTotal.key)}}
+                  </td>
+                  <td>
+                    {{round (get this.saturdayAutoVehicleTripCalcs inOutOrTotal.key)}}
+                  </td>
+                  <td>
+                    {{#if ta.taxiVehicleOccupancy}}
+                      {{round (get this.saturdayTaxiVehicleTripCalcs inOutOrTotal.key)}}
+                    {{else}}
+                      -
+                    {{/if}}
+                  </td>
+                  <td>
+                    -
+                  </td>
+                  <td>
+                    {{#if ta.taxiVehicleOccupancy}}
+                      {{round (get this.saturdayVehicleTripTotals inOutOrTotal.key)}}
+                    {{else}}
+                      -
+                    {{/if}}
+                  </td>
+                {{else}}
+                  {{#each this.modalSplitVariablesSubset as |mode|}}
+                    <td>
+                      {{round (get this.totalsOfSaturdayMode mode)}}
+                    </td>
+                  {{/each}}
+                  <td>
+                    {{round this.totalOfTotalsOfSaturdayMode}}
+                  </td>
+                  <td>
+                    {{round this.totalOfSaturdayAutoVehicleTrip}}
+                  </td>
+                  <td>
+                    {{#if ta.taxiVehicleOccupancy}}
+                      {{round this.totalOfSaturdayTaxiVehicleTrip}}
+                    {{else}}
+                      -
+                    {{/if}}
+                  </td>
+                  <td>
+                    -
+                  </td>
+                  <td>
+                    {{#if ta.taxiVehicleOccupancy}}
+                      {{round this.totalOfTotalsOfSaturdayVehicleTrip}}
+                    {{else}}
+                      -
+                    {{/if}}
+                  </td>
+                {{/if}}
```

Comment:
@godfreyyeung similarly here, was this mostly template formatting / shape / white space that was breaking a template lint rule?

Reply:
Not just formatting. It was primarily deleting two unnecessary `#with` blocks which wrapped this entire section. Since I removed the wrapping `#with`, these lines of code needed to be shifted left two tabs, which is what triggered the massive commit diff .